src/ex5/server/db/migrations/20220618211349-task_id_set_to_PK.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        await queryInterface.changeColumn("Tasks", "id", {type: Sequelize.INTEGER, primaryKey: true });
```

Comment:
Isn't it already intitialized as primary ket in src/ex5/server/db/migrations/20220618195035-create-task.js:8 ?